profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/Datadog.Profiler.Native.Windows.vcxproj

```diff
@@ -266,10 +259,4 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```

Comment:
Was this the _only_ nuget requirement we had on the native side do you know? Do any of the other native projects have nuget deps? 🤔

Reply:
the profiler vcxproj (test, static lib and dll) files have libdatadog as nuget deps.
The test has googletest too.